### ISSUE TITLE
Microsoft FBL messages with "from" addresses surrounded by angle brackets slip through

### ISSUE THREAD

**Issue by lewa** [sisimai/p5-Sisimai#304]
```
From: <staff@hotmail.com>
Subject: complaint about message from 192.0.2.1
```

$mfrom regexp in Sisimai::ARF->is_arf() does not match addresses surrounded by angle brackets. Is this a bug or a feature?

```perl
# Microsoft (Hotmail, MSN, Live, Outlook) uses its own report format.
# Amazon SES Complaints bounces
my $title = 'complaint about message from ';
my $mfrom = qr{(?:
     staff[@]hotmail[.]com
    |complaints[@]email-abuse[.]amazonses[.]com
    )\z
}x;
```

Sisimai v4.24.1

**Comment by azumakuniyuki** [comment]
Thanks for the bug report. This issue will be fixed until the next release (v4.24.1p4).

**Comment by lewa** [comment]
Perfect, thanks. I patched with `>?` on my end in the meantime until the release.

**Comment by azumakuniyuki** [comment]
@lewa Hello, sorry for lately response.
We have released `v4.24.1p5` including code to fix this issue at https://github.com/sisimai/p5-Sisimai/tree/v4.24.1p5.

Thanks,

**Comment by lewa** [comment]
Works like a charm. Thank you!
